Core/Sources/Implementations/species.cpp

```diff
@@ -71,13 +71,36 @@ auto Hippocrates::Species::IsStagnant() const -> bool {
 
 auto Species::LetPopulationLive() -> void {
 	for (auto& organism : population) {
-		while (!organism.HasFinishedTask()) {
-			organism.Update();
-		}
+		UpdateOrganism(organism);
 	}
+
 	isSortedByFitness = false;
 }
 
+auto Species::UpdateOrganism(Organism& organism) -> void {
```

Comment:
The difference between
`organism.Update()`
and
`UpdateOrganism(organism)`
is not evident. 
It looks like the same but does different things